Electronics/PCB/AstroPlant_ExtensionBoard/astroplant-extension-board-mk0-6-1/README.md

```diff
@@ -0,0 +1,42 @@
+
+#If you want to generate output files, follow these steps: 
+
+First you'll need Kicad 7.0: https://www.kicad.org/download/ 
+Then open the project by executing the .kicad_pro file, you will then be able to see the different elements of the project such as the schematic and the pcb,
+If you want the gerber or drill map, open the pcb editor (.kicad_pcb) and go "File -> Fabrication output ->Gerbers/Drill/BOM".
+
+For Gerber, select layers: B_Cu,B_Mask,B_Paste,B_Silkscreen,F_Cu,F_Mask,F_Paste,F_Silkscreen,Edge_Cuts and User_Eco2
```

Comment:
Maybe also mention output folder, it goes to a different location on my laptop, maybe it's something you can adjust in the project file?

Reply:
done